test_scripts/API/VehicleData/UnsubscribeVehicleData/003_RPC_parameter_already_unsubscribed_Result_IGNORED_flow.lua

```diff
@@ -0,0 +1,79 @@
+---------------------------------------------------------------------------------------------------
+-- Item: Use Case: request is allowed by Policies
+--
```

Comment:
@AByzhynar, add link to the requirement when they will be available.

Reply:
@AStasiuk Fixed in 3b703abdab25b3e13b150954e8a8c707f40a3b45